libcob/java.h

```diff
@@ -0,0 +1,50 @@
+/*
+   Copyright (C) 2024 Free Software Foundation, Inc.
+   Written by Simon Sobisch, Nicolas Berthier, Vedant Tewari
+
+   This file is part of GnuCOBOL.
+
+   The GnuCOBOL runtime library is free software: you can redistribute it
+   and/or modify it under the terms of the GNU Lesser General Public License
+   as published by the Free Software Foundation, either version 3 of the
+   License, or (at your option) any later version.
+
+   GnuCOBOL is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public License
+   along with GnuCOBOL.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef COB_JAVA_H
+#define COB_JAVA_H
+
+#include "config.h"
+#include "common.h"
```

Comment:
inclusion guard COB_JAVA_H should not be necessary, given its inclusion in only C files, right?
Please include config.h. + common.h in the including sources before the `#include jjava.h`, not in this header

Reply:
Hum… for the moment it is included  from `common.h` as well.

Reply:
In the end I simply removed `java.h`.